tests/cases/fourslash/autoImportTypeImport1.ts

```diff
@@ -0,0 +1,41 @@
+/// <reference path="fourslash.ts" />
+
+// @verbatimModuleSyntax: true
+// @target: esnext
+
+// @Filename: /foo.ts
+//// export const A = 1;
+//// export type B = { x: number };
+//// export type C = 1;
+//// export class D = { y: string };
+
+// @Filename: /test.ts
+//// import { A, D, type C } from './foo';
+//// const b: B/**/ | C;
+//// console.log(A, D);
+
+goTo.marker("");
+
+verify.importFixAtPosition([
+`import { A, D, type C, type B } from './foo';
+const b: B | C;
+console.log(A, D);`],
+    /*errorCode*/ undefined,
+    { organizeImportsTypeOrder: "inline" }
+);
+
+verify.importFixAtPosition([
+`import { A, D, type C, type B } from './foo';
+const b: B | C;
+console.log(A, D);`],
+    /*errorCode*/ undefined,
+    { organizeImportsTypeOrder: "last" }
+);

```

Comment:
These both seem to have the wrong behavior, right? In each case, prior to the import, the specifiers are correctly sorted according to the preference, but afterwards, they’re not?

Reply:
Yeah, noticed this too; I assumed this was just a starting point for further improvements.

Reply:
`inline` looks alright to me since `A, D, C` is unsorted, but I agree (and am looking into) why `last` is funky. I wasn't sure if `first` should have inserted `type B` at the beginning.

Reply:
I also should note that the previous behavior for auto-imports (which should be equivalent to `last`) also inserted `type B` at the end

Reply:
I think `first` could insert type-only specifiers into unsorted lists at the beginning, but I understand why it didn’t. You’re right that `inline` was also unsorted. Just `last` looks suspicious here.